cmd/server.go

```diff
@@ -99,23 +108,33 @@ func produceServerCmdRunFunc(o serverOperator, action int) cmdRunFunc {
 func initServerCmd() {
 	var serverCmd = &cobra.Command{
 		Use:   "server",
-		Short: "Print server list",
-		Long:  `Print all server information`,
-		Run:   produceServerCmdRunFunc(client, serverMainAction),
+		Short: "Operations on servers",
+		Long:  `List, create, or remove servers.`,
 	}
 
-	var onCmd, offCmd = &cobra.Command{
-		Use:   "on",
+	var serverLsCmd = &cobra.Command{
+		Use:     "ls [flags]",
+		Aliases: []string{"list"},
+		Short:   "List servers",
+		Long:    `List server objects.`,
+		Run:     produceServerCmdRunFunc(client, serverListAction),
+	}
+
+	var serverOnCmd = &cobra.Command{
+		Use:   "on [ID]",
```

Comment:
Oh i like the way you implemented it, i had a quiet similar build locally which i already did for my upcomming Tab-Completion 👍